iothub_client/devdoc/requirement_docs/iothub_client_streaming_requirements.md

```diff
@@ -25,17 +25,17 @@ typedef struct DEVICE_STREAM_C2D_RESPONSE_TAG
 
 typedef DEVICE_STREAM_C2D_RESPONSE* (*DEVICE_STREAM_C2D_REQUEST_CALLBACK)(DEVICE_STREAM_C2D_REQUEST* request, const void* context);
 
-extern DEVICE_STREAM_C2D_RESPONSE* stream_c2d_response_create(DEVICE_STREAM_C2D_REQUEST* request, bool accept, char* data, char* content_type, char* content_encoding);
-extern DEVICE_STREAM_C2D_REQUEST* stream_c2d_request_clone(DEVICE_STREAM_C2D_REQUEST* request);
-extern void stream_c2d_response_destroy(DEVICE_STREAM_C2D_RESPONSE* response);
-extern void stream_c2d_request_destroy(DEVICE_STREAM_C2D_REQUEST* request);
+extern DEVICE_STREAM_C2D_RESPONSE* IoTHubClient_StreamC2DResponseCreate(DEVICE_STREAM_C2D_REQUEST* request, bool accept, char* data, char* content_type, char* content_encoding);
+extern DEVICE_STREAM_C2D_REQUEST* IoTHubClient_StreamC2DRequestClone(DEVICE_STREAM_C2D_REQUEST* request);
+extern void IoTHubClient_StreamC2DResponseDestroy(DEVICE_STREAM_C2D_RESPONSE* response);
+extern void IoTHubClient_StreamC2DRequestDestroy(DEVICE_STREAM_C2D_REQUEST* request);
```

Comment:
Isn't it breaking change?

Reply:
Normally yes, but this is a public preview feature.